### DISCUSSION TITLE
Escala: A web-based score editor

### DISCUSSION THREAD

**Discussion by gallegretti** [CoderLine/alphaTab#1141] [Show and tell]
Hello,

For my final graduation project, I created Escala: A web-based score editor, using AlphaTab. The code is available [here on Github](https://github.com/gallegretti/escala) and you can try it out [here](https://gallegretti.github.io/escala/).

It still has many bugs and shortcomings, but has a nice set of features for editing notes and beats. I think right now it's a good tool for quickly editing existing scores, but not soo much for creating from scratch, as it still lacks some functionalities for that.

Let me know what you think.

**Comment by Danielku15** [comment]
Thanks a lot for sharing this. It makes me happy to see how people use alphaTab for their projects. Works really well and I bet it can act as reference for quite many things. 

What level of graduation did you complete with this project if I may ask?

**Reply by gallegretti** [reply]
Thanks for the support! My degree is the equivalent of an undergraduate degree.
